porcupine/plugins/anchors.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
```

Comment:
Please add a docstring to the beginning of the file. It will show up in plugin manager.